apps/web/ui/dub-partners-logo.tsx

```
@@ -0,0 +1,15 @@
+import { Wordmark } from "@dub/ui";
+import { cn } from "@dub/utils";
+
+export function DubPartnersLogo({ className }: { className?: string }) {
+  return (
+    <a
+      href="https://dub.co/partners"
+      target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)